src/main/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionController.java

```diff
@@ -164,7 +167,7 @@ private boolean transcriptionTypesThatRequireDates(Integer transcriptionTypeId)
     }
 
     private boolean transcriptionDatesAreSet(OffsetDateTime startDateTime, OffsetDateTime endDateTime) {
-        return !isNull(startDateTime) || !isNull(endDateTime);
+        return !isNull(startDateTime) && !isNull(endDateTime);
```

Comment:
I guess `nonNull` like in method validateTranscriptionRequestValues might be cleaner if needing to use Objects at all